adrs/0002-monorepo.md

```diff
@@ -0,0 +1,34 @@
+# [Transition Stacks to a monorepo]
+
+**Date:** 2022-11-28
+
+**Related PR:** #1191
+
+## Issue
+
+The Stacks repo is growing in complexity and will soon require many different dependencies that are optional for most consumers.
+
+## Decision
+
+Despite shipping many different concepts (atomic classes, CSS components, JS components), the interdependency chains are relatively linear. This means that we can fairly trivially split the repo into smaller portions.
+
+Our current codebase could be split like so:
+- Atomic classes (no dependencies)
+- CSS Components (depends on Atomic components)
+- JS Components (depends on Stimulus, Popper.js, CSS + Atomic components)
+- Documentation site (depends on JS components, 11ty + plugins)
```

Comment:
if we keep js and css components in the same package then this will depends on a "stacks-components" package and transitively on the atomic classes package (perhaps "stacks-base")